api/src/main/java/com/datastrato/gravitino/authorization/SecurableObjects.java

```diff
@@ -102,8 +102,8 @@ public static SecurableObject ofFileset(SecurableObject schema, String fileset)
   /**
    * All metalakes is a special securable object .You can give the securable object the privileges
    * `CREATE METALAKE`, etc. It means that you can create any which doesn't exist. This securable
-   * object is only used for metalake admin. You can't grant any privilege to this securable object.
-   * You can't bind this securable object to any role, too.
+   * object is reserved for system. You can't grant any privilege to this securable object. You
+   * can't bind this securable object to any role, too.
```

Comment:
There you mentioned that "You can give the securable object the privileges `CREATE METALAKE`", but here you mentioned that "You can't grant any privilege to this securable object". I think these two sentences are self-contradictory, you should have a more accurate description.

Besides, why you can't bind this securable object to a role?

Reply:
Yes, for users, users can't grant any privilege to allMetalakes. But system pre-created a role binds allMetalakes. AllMetalakes isn't used for the normal user. 

We shouldn't allow user to use allMetalakes. It can inflluence the privileges of all metalakes.It's too dangerous.